grammar.js

```diff
@@ -4,19 +4,18 @@ const PHP = require('tree-sitter-php/grammar');
 // TODO array return types https://docs.phpdoc.org/3.0/guide/references/phpdoc/types.html#arrays
 // PHPDoc tags: https://docs.phpdoc.org/3.0/guide/references/phpdoc/tags/index.html#tag-reference
 // PHPDoc docs appear to use these conventions
+// TODO inline tags in inline tags (allowed for @internal)
 //   @verbatimElement [required element] [<optional element>]
 
 module.exports = grammar({
   name: 'phpdoc',
 
   extras: $ => [
-    token(
-      seq(/(\r)|(\r\n)|(\n)/, /[ \t]*/, repeat('*'), /[ \t]*/)
-    ),
-    token(
-      seq(/[ \t]*/, repeat('*'), /[ \t]*/)
-    ),
-    token(repeat1(' ')),
```

Comment:
I've replace this with the original whitespace regex again. I think we could even use `/\s/` instead. I've seen that in other parsers.

Reply:
Sounds good to me.